tests/jest/db/DAO.spec.js

```diff
@@ -426,6 +424,7 @@ test('StudyRequestDAO', async () => {
     status: 'REQUESTED',
     serviceRequestId: null,
     priority: 'STANDARD',
+    assignedTo: null,
```

Comment:
please run this test to ensure that it still works (it's not part of pre-commit checks, since it's quite slow)